src/WebJobs.Script/Workers/WorkerConstants.cs

```diff
@@ -29,6 +29,13 @@ public static class WorkerConstants
 
         // Profiles
         public const string WorkerDescriptionProfiles = "profiles";
+        public const string WorkerDescriptionProfileName = "name";
```

Comment:
The value `"name"` here is same for `WorkerDescriptionProfileName` and `WorkerDescriptionProfileConditionName`. Can we assign them values - `profileName` and `profileConditionName` respectively to make them more descriptive? This also applies to other newly added variables in this file.